src/jade/tabs/advanced.jade

```diff
@@ -8,6 +8,9 @@ section.col-xs-12.content(ng-controller="AdvancedCtrl")
       .alert.alert-success(ng-show="success.saveBlob", l10n) Your blob vault has been changed successfully.
       .alert.alert-success(ng-show="success.saveBridge", l10n) Your bitcoin bridge has been changed successfully.
       .alert.alert-success(ng-show="success.saveAcctOptions", l10n) Trustline advanced settings has been changed successfully.
+      .alert.alert-success(ng-show="success.saveConfirmation.send", l10n) Send confirmation settings has been changed successfully.
```

Comment:
'settings' is plural. Would the correct form for this and the two following strings not be '[xyz] settings **have** been changed'?